src/java.desktop/unix/native/libawt_xawt/awt/awt_GraphicsEnv.c

```diff
@@ -1287,7 +1291,7 @@ Java_sun_awt_X11GraphicsDevice_pGetBounds(JNIEnv *env, jobject this, jint screen
             AWT_UNLOCK ();
 
             bounds = (*env)->NewObject(env, clazz, mid, 0, 0,
-                    xwa.width, xwa.height);
+                                    xwa.width, xwa.height);
```

Comment:
I'm for reverting this change. The existing formatting was consistent in indenting wrapped lines by 8 spaces. The new formatting doesn't stick to either style: not 8 spaces, nor aligned to the opening parenthesis. The former is used in the file, see lines 1289–1290.

Reply:
Reverted.